mycroft/skills/__main__.py

```diff
@@ -81,20 +81,17 @@ def prepare_device(self):
         """Internet dependent updates of various aspects of the device."""
         self._get_pairing_status()
         self._update_system_clock()
-        if self.time_skew_threshold_exceeded:
-            self._reboot()
-        else:
-            self._update_system()
-            # Above will block during update process and kill this instance if
-            # new software is installed
+        self._update_system()
```

Comment:
You got rid of the only usage of self.time_skew_threshold_exceeded.  Removal of that property should be included in this PR.

Reply:
Thanks. `_reboot()` was also no longer needed and remove. Re-tested and works just fine.